main.py

```diff
@@ -2,6 +2,30 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres_to_create = ["Western", "Action", "Dramma"]
+    for genre in genres_to_create:
+        Genre.objects.create(name=genre)
+
+    actors_to_create = [
+        ("George", "Klooney"), ("Kianu", "Reaves"), ("Scarlett", "Keegan"),
+        ("Will", "Smith"), ("Jaden", "Smith"), ("Scarlett", "Johansson")
+    ]
+    for first_name, last_name in actors_to_create:
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
```

Comment:
Would make sense to break down main function is smaller specialized functions to structure your code better. E.g separate function for creating, updating, deleting..